package.json

```diff
@@ -7,21 +7,21 @@
     "npm": ">=7.5.3"
   },
   "scripts": {
-    "test": "./node_modules/.bin/cypress-tags run -eTAGS=@cypress"
+    "test": "./node_modules/.bin/cypress-tags run -eTAGS=@cypress",
+    "cypress:open": "npx cypress open"
```

Comment:
`npx` is redundant in this line, e.g., `cypress open` should be sufficient. (The same applies to the above line where `cypress-tags run -eTAGS=@cypress` should be sufficient.